README.md

```diff
@@ -55,6 +55,30 @@ import base64
 base64.b32encode(os.urandom(10))
 ```
 
+#### Reading the secret from client service's infrastructure code
+
+Once the service's secret is stored in Azure Key Vault, it can be retrieved
+from there in S2S infrastructure (Terraform) code. In order to avoid duplication,
+we recommend that the infrastructure definition of the client service also reads
+this secret. Here's some Terraform code that does it:
+
+```
+data "azurerm_key_vault_secret" "microservice_key" {
```

Comment:
maybe do `s2s_key` microservice_key probably doesn't make sense outside of this repo?